### ISSUE TITLE
If 97 examples VS20017

### ISSUE THREAD

**Issue by IamBelka** [CoolProp/IF97#32]
Good afternoon, I came across your implementation of the IF-97 method and faced some difficulties when porting the code to the vs2017 environment. I create a blank project, connected the files IF97.cpp and IF97.h. The project compiles, but the tables do not map their contents to the IAPWS document when they are displayed. Prompt in what there can be a mistake?
![_97](https://user-images.githubusercontent.com/46045560/50335908-d78c5700-051d-11e9-917d-3db8dd732324.png)

**Comment by henningjp** [comment]
This is odd.  I've verified this on VS2010, VS2015, and VS2017.  I just compiled the latest version on VS2015 and got this:
![if97 test](https://user-images.githubusercontent.com/17114032/50340764-4d76cb00-04ea-11e9-862b-e28d0ef7ec12.png)

I won't have access to my VS2017 software/machine until after the new year, but I suspect I'll get the same answers there.

First question, did you change anything at all in the code before compiling it?

Second, did you build your VS project using the cmake script provided in the README file on the Github CoolProp/IF97 page (https://github.com/CoolProp/IF97)?
```
mkdir build
cd build
cmake ..
cmake --build .
```
